### PR TITLE
update_engine: update live ebuild for current master

### PR DESCRIPTION
**Pull request by marineam**

### CODE REVIEW

File: coreos-base/update_engine/update_engine-9999.ebuild [hunk 1]
```diff
@@ -5,13 +5,11 @@ EAPI=5
...
 CROS_WORKON_REPO="git://github.com"
 AUTOTOOLS_AUTORECONF=1
-# TODO: this can be disabled once -I.. is no longer used
-AUTOTOOLS_IN_SOURCE_BUILD=1
 
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="c88492c39ec0ad81bb26ae6efde7b4fc46a9804a"
+	CROS_WORKON_COMMIT="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
```
Comment (crawford) [inline coreos-base/update_engine/update_engine-9999.ebuild]: Any reason not to just delete this line?

File: profiles/coreos/base/package.use [hunk 2]
```diff
@@ -72,3 +72,6 @@ sys-auth/sssd -manpages -python
 
 # needed for realmd build
 sys-auth/polkit introspection
+
+# enable logging command-line options in update_engine
+dev-cpp/glog gflags
```
Comment (crawford) [inline profiles/coreos/base/package.use]: Won't this affect the stable ebuild as well?

Comment (marineam) [inline profiles/coreos/base/package.use]: No, it doesn't use glog

File: coreos-base/update_engine/update_engine-9999.ebuild (hunk 1 above)
Comment (marineam) [inline coreos-base/update_engine/update_engine-9999.ebuild]: it'll be added back with a real hash soon but I'll prune for now.

**Comment by marineam** [comment]
Updated, dropping the unnecessary fake git hash and the invalid stable arm64 on the old version, the ebuild currently isn't built for arm.

**Comment by crawford** [comment]
LGTM

**Pull request merged**